core/frontend/src/components/ethernet/AddressDeletionDialog.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-dialog
+    width="350"
+    :value="show"
+    @input="resolve"
+  >
+    <v-card>
+      <v-card-title>{{ title }}</v-card-title>
+
+      <v-card-subtitle class="mb-3 mt-3">
+        {{ message }}
+      </v-card-subtitle>
+
+      <v-card-text class="d-flex flex-row justify-space-around">
+        <v-btn
+          color="grey"
+          class="ma-2 elevation-2"
+          text
+          @click="resolve(false)"
+        >
+          Abort
+        </v-btn>
+        <v-btn
+          color="primary"
+          class="ma-2 elevation-2"
+          text
+          @click="resolve(true)"
+        >
+          Confirm
+        </v-btn>
+      </v-card-text>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
+  name: 'AddressDeletionDialog',
+  model: {
+    prop: 'show',
+    event: 'change',
+  },
+  props: {
+    show: {
+      type: Boolean,
+      default: false,
+    },
+    dialogType: {
+      type: String,
+      default: '',
+      validator: (value: string) => ['', 'last-ip-address', 'ip-being-used'].includes(value),
+    },
+  },
+  data() {
+    return {
+      resolveCallback: undefined as ((state: boolean) => void) | undefined,
+    }
+  },
+  computed: {
+    title() {
+      if (this.dialogType === 'last-ip-address') {
+        return 'Last IP Address'
+      }
+      if (this.dialogType === 'ip-being-used') {
+        return 'IP Address in Use'
+      }
+
+      return 'IP Deletion'
+    },
+    message() {
+      if (this.dialogType === 'last-ip-address') {
+        return 'This is the last IP address on the interface. Are you sure you want to proceed?'
+      }
+      if (this.dialogType === 'ip-being-used') {
+        return 'The IP address is currently being used to access BlueOS. Are you sure you want to proceed?'
```

Comment:
```suggestion
        return 'The IP address is currently being used to access BlueOS. Deleting it could prevent you from accessing your vehicle. Are you sure you want to proceed?'
```